.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Please remove this file, instead, you could include `.vscode` in the .gitignore file